README.md

```diff
@@ -7,17 +7,60 @@
 - Web: Firefox, Chromium
 - Larger mobile devices like tablets: Progressive Web App (PWA) Android 9+
 
+## Documentation
+
 Start reading in [/doc/architecture/README.md](/doc/architecture/README.md).
 The documentation is best viewed by running the following commands in the projects root folder:
 
 ```sh
-cargo install mdbook mdbook-mermaid
-cargo install --git https://github.com/ElektraInitiative/mdbook-generate-summary mdbook-generate-summary
-mdbook serve --open
+make run-mdbook
 ```
 
 Which will open [/doc/architecture/README.md](/doc/architecture/README.md) as first page.
 
+## Quickstart
+
+Start the frontend and backend in two seperate shells.
+
+```sh
+make run-frontend
+```
+
+```sh
+make run-backend
+```
+
+Visit http://localhost:8080
```

Comment:
Better to remove this, as it won't work for someone trying PermaplanT the first time

```suggestion
```